### PR TITLE
feat: automod changes

### PR DESCRIPTION
**Pull request by Chicken**
Opionated changes to the wordlist and modified functionality for less strict automod.

### CODE REVIEW

**Comment by KommentatorForAll** [comment]
closes #171

File: src/modules/automod/message.js [hunk 1]
```diff
@@ -10,35 +10,20 @@ import config from '../../config';
...
-        .replaceAll(/[0ö]/g, 'o')
-        .replaceAll(/[1!|]/g, 'i')
-        .replaceAll(/3/g, 'e')
-        .replaceAll(/[4@ä]/g, 'a')
-        .replaceAll(/[5$]/g, 's')
-        .replaceAll(/[7+]/g, 't')
-        .replaceAll(/8/g, 'b')
-        .replaceAll(/\(/g, 'c')
-        .replaceAll(/ü/g, 'u')
-        .replaceAll(/[^a-z ]/g, '');
```
Comment (WilsontheWolf) [inline src/modules/automod/message.js:27]: Do we want to remove this. Without regex this won't cause false positives.

File: src/modules/automod/message.js [hunk 2]
```diff
@@ -10,35 +10,20 @@ import config from '../../config';
...
-    const regex = Blacklist.swearRegex;
-    return content.match(regex);
+    const words = message.content.toLowerCase().split(' ');
```
Comment (WilsontheWolf) [inline src/modules/automod/message.js:16]: splitting by `/(?:\b|\s)+/g` might be smarter.
435F7E2F-F93A-4CA8-8812-89A1B4472471

**Review by WilsontheWolf — CHANGES_REQUESTED** [review]

File: src/modules/automod/message.js (hunk 1 above)
Comment (Chicken) [reply src/modules/automod/message.js:27]: Yes, point of this is to just keep the thing simplistic

**Review by WilsontheWolf — APPROVED** [review]

**Review by KommentatorForAll — APPROVED** [review]

**Comment by Chicken** [comment]
dont merge k ty

**Pull request merged**